faq/faq.md

```diff
@@ -0,0 +1,351 @@
+parts of Firefox do.
+
+If all the code within the file is licensed under multiple licenses, and the
+user must comply with all licenses simultaneously, use
+`SPDX-License-Identifier: LGPL-2.0-or-later AND AML`, as can be found in Simple
+DirectMedia Layer (SDL).
+
+If all the code within the file is licensed under either one license or another
+(for instance, all code is under GPL-2.0-only, but one function is under MIT),
+use separate tags `SPDX-License-Identifier: GPL-2.0-only` and
+`SPDX-License-Identifier: MIT`.
+
+You can read more about SPDX expressions [on the SPDX
+wiki](https://wiki.spdx.org/view/LicenseExpressionFAQ).
+
+## How to deal with MIT/BSD licenses which include copyright information themselves? {#mit-bsd}
+
+TODO: Figure this one out
+
+## Should I move my existing LICENSE or COPYING file? {#move-license}
+
+If you use multiple licenses in your project, we recommend that you put all of
+your licenses in the `LICENSES/` directory.
+
+If you only have a single license file, see [the next
+question](#single-license).
+
+## I only have a single license file. Should I still create a LICENSES directory? {#single-license}
+
+You do not strictly need to put your license in the `LICENSES/` directory,
+though we recommend that you do. If you want to keep your license in a single
+file (`LICEN[CS]E`, `COPYING`, `COPYRIGHT`), you should edit the file to signal
+what licence you are actually using.
+
+Preface your license with the following header:
+
+```
+Valid-License-Identifier: [identifier]
+License-Text:
+
+[license text]
+```
+
+## Should I put comment headers in my license files? {#header-in-license}
+
+## How do I use a license that is not on the SPDX License List? {#custom-license}
+
+<!-- TODO: Explain that the user probably shouldn't use a custom license -->
+
+If you have a custom or modified license that does not appear in the SPDX
+License List, place your license in the file
+`LICENSES/LicenseRef-MyLicense.txt`, and/or preface your license text with the
+tag `Valid-License-Identifier: LicenseRef-MyLicense`. By doing this, SPDX will
+still be able to recognise your license.
+
+## Should I edit my license files? {#edit-license}
+
+You should never edit license files. When you use an existing license, you
+should always copy it verbatim.
+
+Some licenses, such as MIT and the BSD family of licenses, have a line that says
+"Copyright (c) [year] [copyright holder]". Please see [this question](#mit-bsd)
+about how to deal with those licenses.
+
+There are many reasons for why you should not alter license texts, but if you do
+alter the texts, you should use a different SPDX identifier for this license.
+See [the previous question](#custom-license).
+
+## Can I edit copyright and licensing tags? {#edit-tags}
+
+TODO: If erroneous, sure.
+
+## Can I remove the license and copyright information from minified code (e.g., JavaScript)? {#minified}
+
+Probably shouldn't
+
+(but REUSE only applies to source repositories)
```

Comment:
Modern minifiers/uglifiers have flags to retain licensing and copyright info. Make sure to use those flags and preserve the info.

In those cases, you save *much* more space by using REUSE on that code and retaining only these two lines than if you were using a full traditional header.

Reply:
Fixed in a642d2114f67b7a7a68b13afa9a965a7a3325f41